### ISSUE TITLE
Uncaught ReferenceError: Ps is not defined

### ISSUE THREAD

**Issue by bencoullie** [YourCursus/perfect-scrollbar-rails#1]
Hi there,

I've done everything asked of me (added gem to gemfile, bundled, added require to CSS and JS application files). However when I run the rails app I get the Uncaught ReferenceError: Ps is not defined which I'm assuming means that the rails engine was unable to deliver the perfect scrollbar JS requirement - any ideas?

If you need any other information just ask.

Cheers,

**Comment by zedtux** [comment]
Hello @bencoullie,

Can you please post the entire backtrace ?

**Comment by pablobarria** [comment]
Hi,

I'm having the exact same issue. There's no backtrace because nothing fails on Rails. But the Ps object is not available in JavaScript. Is there an additional step besides installing the gem and requiring perfect-scrollbar on application.js?

thanks.

**Comment by pablobarria** [comment]
...answering my own question: I had removed jquery as a dependency in my application.
I think perfect-scrollbar-rails assumes that the perfect-scrollbar jquery plugin is being used. If I replace the non jquery version of the plugin in vendor/assests/javascript, it works.

You should probably add a warning about that.

cheers,
Pablo.

**Comment by zedtux** [comment]
@pablobarria thank you for your messages. So you meant that you're developing a Rails app without using jQuery and in this case the gem is not working?

**Comment by pablobarria** [comment]
Exactly. I decided to turn jQuery off since I was using React as my main JavaScript library. They are not incompatible, but I like to keep a low dependency count. 

I'm not a 100% sure that not using jQuery in rails is a good idea, so rather than go the extra mile and make both versions available, I'd suggest you just warn people about it on your Readme file.

thanks,
P.

**Comment by zedtux** [comment]
@pablobarria can you please then make a PR with the `README.md` update you're expecting?
